app/code/Magento/Store/Test/Unit/Model/StoreTest.php

```diff
@@ -162,6 +162,25 @@ public function testSetWebsite()
         $this->assertEquals(2, $model->getWebsiteId());
     }
 
+    /**
+     * @return void
+     */
+    public function testGetId()
+    {
+        $testId = 1;
+        $websiteRepository = $this->getMockBuilder(WebsiteRepositoryInterface::class)
+            ->getMockForAbstractClass();
+
+        /** @var Store $model */
+        $model = $this->objectManagerHelper->getObject(
+            Store::class,
+            ['websiteRepository' => $websiteRepository]
+        );
```

Comment:
Please avoid to use ObjectManager helper in the new tests, because it's deprecated.

Reply:
Thanks for your feedback. Fixed.